src/custom-components/ReorderableList.tsx

```diff
@@ -0,0 +1,127 @@
+import { Fragment, JSXElementConstructor, ReactElement, useEffect, useState } from "react";
+import { Field, FieldProps, Focusable, GamepadButton } from "../deck-components";
+
+export type ReorderableEntry<T> = {
+  label: string,
+  data?:T,
+  position:number
+}
+
+type ListProps<T> = {
+  entries: ReorderableEntry<T>[],
+  onSave: (entries: ReorderableEntry<T>[]) => void,
+  interactables?: JSXElementConstructor<{entry:ReorderableEntry<T>}>,
+  fieldProps?: FieldProps
+}
+
+/**
+ * A component for creating reorderable lists.
+ * 
+ * Implementation example can be found {@link https://github.com/Tormak9970/Component-Testing-Plugin/blob/main/src/testing-window/ReorderableListTest.tsx here}.
```

Comment:
use `@example` for better doc

Reply:
You want me to put the example in the jsDoc comment?